scripts/config/msu/text.nut

```diff
@@ -13,4 +13,32 @@
 	{
 		return this.color(::Const.UI.Color.NegativeValue, _string);
 	}
+
+	function colorizeValue( _value, _compareTo = 0, _addSign = true, _invertColor = false, _addPercent = false )
+	{
+		if (_value < _compareTo)
+		{
+			if (!_addSign) _value *= -1;
+			if (_addPercent) _value = _value + "%";
```

Comment:
_addPercent seems repeated four times, just do it at the beginning

Reply:
Ah I see you can't do that due to the negative case. Might want to define a return string and manipulate that instead, but doesn't matter much,